src/Core/Prefab/Globe/GlobeLayer.js

```diff
@@ -73,6 +116,108 @@ class GlobeLayer extends TiledGeometryLayer {
         }
         return occupancy;
     }
+
+    // eslint-disable-next-line
+    culling(node, camera) {
+        if (!camera.isBox3Visible(node.OBB().box3D, node.OBB().matrixWorld)) {
+            return true;
+        }
+
+        if (node.level < 2) {
+            return false;
+        }
+
+        const points = node.OBB().topPointsWorld;
+        const magnitudeSquared = cameraPosition.lengthSq() - 1.0;
```

Comment:
could you move `magnitudeSquared` in preUpdate

Reply:
Done